sdk/src/test/resources/SidechainBlockTestVrfProof908

```diff
@@ -1 +1 @@
-ed052dfda272eae6acf4756ac61dc7a4c187da38390cda0490af5a133d8c9d3700bd0e22e9dcde68a5c603e6a181cfeefc0c6a02972cfdd992aa3893cc9af24729bde88af6c55448df495021397338532e66017dc204b7faeff85560f6106ed026
\ No newline at end of file
+3ae9384dac6b58a04487851010b71b56c4dcd9d33dcec12ba07d8205bc89570c80380ca2319ad37fbf1a2a07fb7435de4aa5a40400d2d66c1d89f3c4e9b5029a043048d6df79efaa8e7e981533ca66ffdc7cd9ae26a1b1fc9581e048b624af5738
```

Comment:
what was the cause to change this test file and 2 others below: sdk/src/test/resources/SidechainBlockTestVrfSecretKey908 and sdk/src/test/resources/sidechainblock_hex ?

Reply:
The secret key witch was used to generate this proof has some heading zeros, according to new key validation we cannot use it any more. So we had to regenerate secret key. proof and the sidechain block.